common/src/stack/storage-config/bin/configure-controllers.py

```diff
@@ -181,11 +176,15 @@ def getController():
 		#
 		continue
 
-	if raidlevel == 'hotspare' and arrayid == 'global':
-		ctrl.doGlobalHotSpare(adapter, enclosure, hotspares, options)
-	else:
-		ctrl.doRaid(raidlevel, adapter, enclosure, slots, hotspares,
-			options)
+	try:
+		if raidlevel == 'hotspare' and arrayid == 'global':
```

Comment:
Not a bad thought, but I'd be concerned that it would require int()-coercion (I'm not sure that the rest of the code treats this as a string).  The case-ness looks like it's enforced on the "input-side" in the stack `add` commands.